docs/tutorial/impacts.md

```diff
@@ -39,19 +39,15 @@ You can extract an impact with:
 
 ```python
 >>> response.impacts.usage.gwp.value  # (1)!
-0.34    # Expressed in kgCO2eq.
+RangeValue(min=0.16, max=0.48) # Expressed in kgCO2eq (2)
 ```
 
 1. Assuming you have made an inference and get the response in an `response` object.
+2. [`RangeValue`][utils.range_value.RangeValue] are used to define intervals. It corresponds to the 95% confidence interval of our approximation.
 
-Or you could get **value range** impact instead:
+This range of values corresponds a **high-confidence approximation interval**, within which we are confident enough that the true consumption lies (see the [methodology](../methodology/llm_inference.md) for more information).
```

Comment:
I think we need to change this. It indeed corresponds to the "high-confidence approximation interval", but also to the estimation of model sizes for instance.

Reply:
Agreed, initially that's what I wanted to say, something like "We think it's in this interval for various reasons, including the approximation of the model size AND the error noise of the regression"

I'll rephrase and get back to you